reactioncommerce/reaction
# [WIP] Fix on dashboard analytics settings

[WIP] Resolves #2301 
**Testing**
- Clone this branch
- Follow the directions on the [Reaction docs](https://docs.reactioncommerce.com/reaction-docs/master/analytics) to setup `Analytics` (preferably both `Google` and `Segment.io`)
- Login as admin and try to configure analytics from the dashboard
- Click on the gear icon and observe that the pane for the analytics settings shows
